### PR TITLE
Porting System.Diagnostics source comments

### PR DESCRIPTION
**Pull request by noahfalk**
## Summary

Porting System.Diagnostics triple slash comments

Fixes #54863

cc @tarekgh

### CODE REVIEW

**Comment by noahfalk** [comment]
@carlossanlop

File: xml/System.Diagnostics/Activity.xml [hunk 1]
```diff
@@ -973,10 +974,14 @@ The root ID is a substring from the <xref:System.Diagnostics.Activity.Id> or the
...
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">The baggage key name</param>
+        <param name="value">The baggage value mapped to the input key</param>
+        <summary>Add or update the Activity baggage with the input key and value.
+            If the input value is <see langword="null" /> - if the collection has any baggage with the same key, then this baggage will get removed from the collection.
+                - otherwise, nothing will happen and the collection will not change.
+            If the input value is not <see langword="null" /> - if the collection has any baggage with the same key, then the value mapped to this key will get updated with the new input value.
+                - otherwise, the key and value will get added as a new baggage to the collection.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics/Activity.xml:983]: would make sense moving this section to the remarks? I am ok either way. just checking.

**Comment by tarekgh** [comment]
@noahfalk is it possible to include the Propagators comments too?

File: xml/System.Diagnostics/Activity.xml (hunk 1 above)
Comment (noahfalk) [reply xml/System.Diagnostics/Activity.xml:983]: I'm fine as-is, but if anyone champions that it should be changed I am happy to do so.

File: xml/System.Diagnostics/Activity.xml [hunk 2]
```diff
@@ -973,10 +974,14 @@ The root ID is a substring from the <xref:System.Diagnostics.Activity.Id> or the
...
+        <returns>'this' for convenient chaining</returns>
```
Comment (tarekgh) [inline xml/System.Diagnostics/Activity.xml:984]: > <returns>'this' for convenient chaining</returns> [](http://example.com/codeflow?start=7&length=50)

<returns><see langword="this" /> for convenient chaining.</returns>

**Comment by noahfalk** [comment]
> @noahfalk is it possible to include the Propagators comments too?

I was following the instructions in #54863. I didn't intentionally filter any APIs so I assumed it captured everything that was already populated.

File: xml/System.Diagnostics/Activity.xml [hunk 3]
```diff
@@ -1204,11 +1209,20 @@ This method has the advantage that no string manipulation is needed to set the I
...
       <Docs>
-        <param name="code">To be added.</param>
-        <param name="description">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="code">The status code</param>
+        <param name="description">The error status descrition</param>
+        <summary>Sets the status code and description on the current activity object.</summary>
+        <returns>'this' for convenient chaining</returns>
```
Comment (tarekgh) [inline xml/System.Diagnostics/Activity.xml:1215]: ><returns>'this' for convenient chaining</returns> [](http://example.com/codeflow?start=8&length=49)

<returns><see langword="this" /> for convenient chaining.</returns>

Comment (noahfalk) [reply xml/System.Diagnostics/Activity.xml:1215]: this change is just the results of the automated port tool, I think you are proposing this should be changed in the source?

File: xml/System.Diagnostics/Activity.xml [hunk 4]
```diff
@@ -1400,7 +1414,7 @@ The `StartTimeUtc` property is typically initialized when <xref:System.Diagnosti
         <ReturnType>System.Diagnostics.ActivityStatusCode</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets status code of the current activity object.</summary>
         <value>To be added.</value>
```
Comment (tarekgh) [inline xml/System.Diagnostics/Activity.xml:1418]: ><value>To be added.</value> [](http://example.com/codeflow?start=8&length=27)

<value>Status code of current activity object.</value>

File: xml/System.Diagnostics/Activity.xml [hunk 5]
```diff
@@ -1421,7 +1435,7 @@ The `StartTimeUtc` property is typically initialized when <xref:System.Diagnosti
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the status descrition of the current activity object.</summary>
         <value>To be added.</value>
```
Comment (tarekgh) [inline xml/System.Diagnostics/Activity.xml:1439]: ><value>To be added.</value> [](http://example.com/codeflow?start=8&length=27)

<value>The status description of the current activity object.</value>

File: xml/System.Diagnostics/Activity.xml [hunk 6]
```diff
@@ -1421,7 +1435,7 @@ The `StartTimeUtc` property is typically initialized when <xref:System.Diagnosti
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the status descrition of the current activity object.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics/Activity.xml:1438]: >descrition [](http://example.com/codeflow?start=33&length=10)

description

File: xml/System.Diagnostics.Metrics/Instrument`1.xml [hunk 7]
```diff
@@ -74,8 +83,8 @@
         <Parameter Name="measurement" Type="T" />
       </Parameters>
       <Docs>
-        <param name="measurement">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="measurement">The measurement value.</param>
+        <summary>Record the measurement by notifying all <see cref="T:System.Diagnostics.Metrics.MeterListener" /> objects which listeneing to this instrument.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/Instrument`1.xml:87]: >listeneing [](http://example.com/codeflow?start=129&length=10)

listening

File: xml/System.Diagnostics.Metrics/Instrument`1.xml [hunk 8]
```diff
@@ -99,9 +108,9 @@
         <Parameter Name="tag" Type="System.Collections.Generic.KeyValuePair&lt;System.String,System.Object&gt;" />
       </Parameters>
       <Docs>
-        <param name="measurement">To be added.</param>
-        <param name="tag">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="measurement">The measurement value.</param>
+        <param name="tag">A key-value pair tag associated with the measurement.</param>
+        <summary>Record the measurement by notifying all <see cref="T:System.Diagnostics.Metrics.MeterListener" /> objects which listeneing to this instrument.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/Instrument`1.xml:113]: >listeneing [](http://example.com/codeflow?start=129&length=10)

listening

File: xml/System.Diagnostics.Metrics/Instrument`1.xml [hunk 9]
```diff
@@ -125,9 +134,9 @@
         <Parameter Name="tags" Type="System.ReadOnlySpan&lt;System.Collections.Generic.KeyValuePair&lt;System.String,System.Object&gt;&gt;" />
       </Parameters>
       <Docs>
-        <param name="measurement">To be added.</param>
-        <param name="tags">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="measurement">The measurement value.</param>
+        <param name="tags">A span of key-value pair tags associated with the measurement.</param>
+        <summary>Record the measurement by notifying all <see cref="T:System.Diagnostics.Metrics.MeterListener" /> objects which listeneing to this instrument.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/Instrument`1.xml:139]: >listeneing [](http://example.com/codeflow?start=129&length=10)

listening

File: xml/System.Diagnostics.Metrics/Instrument`1.xml [hunk 10]
```diff
@@ -152,10 +161,10 @@
...
       </Parameters>
       <Docs>
-        <param name="measurement">To be added.</param>
-        <param name="tag1">To be added.</param>
-        <param name="tag2">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="measurement">The measurement value.</param>
+        <param name="tag1">A first key-value pair tag associated with the measurement.</param>
+        <param name="tag2">A second key-value pair tag associated with the measurement.</param>
+        <summary>Record the measurement by notifying all <see cref="T:System.Diagnostics.Metrics.MeterListener" /> objects which listeneing to this instrument.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/Instrument`1.xml:167]: >listeneing [](http://example.com/codeflow?start=129&length=10)

listening

File: xml/System.Diagnostics.Metrics/Instrument`1.xml [hunk 11]
```diff
@@ -181,11 +190,11 @@
...
-        <param name="measurement">To be added.</param>
-        <param name="tag1">To be added.</param>
-        <param name="tag2">To be added.</param>
-        <param name="tag3">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="measurement">The measurement value.</param>
+        <param name="tag1">A first key-value pair tag associated with the measurement.</param>
+        <param name="tag2">A second key-value pair tag associated with the measurement.</param>
+        <param name="tag3">A third key-value pair tag associated with the measurement.</param>
+        <summary>Record the measurement by notifying all <see cref="T:System.Diagnostics.Metrics.MeterListener" /> objects which listeneing to this instrument.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/Instrument`1.xml:197]: >listeneing [](http://example.com/codeflow?start=129&length=10)

listening

File: xml/System.Diagnostics.Metrics/MeterListener.xml [hunk 12]
```diff
@@ -84,7 +84,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Disposes the listeners which will stop it from listeneing to any instrument.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/MeterListener.xml:87]: >listeneing [](http://example.com/codeflow?start=64&length=10)

listening

File: xml/System.Diagnostics.Metrics/MeterListener.xml [hunk 13]
```diff
@@ -227,7 +229,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Enable the listener to start listeneing to instruments measurement recording.</summary>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/MeterListener.xml:232]: >listeneing [](http://example.com/codeflow?start=46&length=10)

listening

**Comment by noahfalk** [comment]
I did a quick spot check and I no longer see the previous mispellings

**Review by tarekgh — APPROVED** [review]

**Comment by noahfalk** [comment]
@tarekgh - I don't have merge rights in this repo. If we need additional reviewers I am happy to wait, otherwise could you click the button for me? Thanks!

**Pull request merged**